data.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "M Hassan",
+    "githubUsername": "Mhassanbughio",
+    "imageUrl": "https://drive.google.com/file/d/1X3n8eob7GXPx9SWRZbIF1hOzDPa5kMop/view?usp=sharing",
```

Comment:
Please can you change this to a screenshot:
### What to do:
- [x] Take a screenshot of your profile, up to where it says `pinned repos`.
- [x] As you just took a screenshot, just `ctrl v` it into the issue. Or, save it to your computer and then upload it to GitHub, and it will generate a custom image URL, when you copy image address.

### Use this and change your image url to the generated one @Mhassanbughio .
Any help, feel free to ask.

Reply:
Also, please can you move you changes to the bottom of the file. thank you